content/en/docs/deployment/releases.md

```diff
@@ -77,6 +77,46 @@ You can build specific releases using `rebar3 release -n <release_name>`
 
 ## Build Configuration
 
+### App Syntax {id="app-syntax"}
+
+```erlang
+{release, {myrelease, "0.1.0"},
+ [<app1>, <app2>]}
+```
+
+In a release, each `<app>` is an atom of the app name, or a tuple with the app name and additional options:
+
+```erlang
+% No options specified
+myapp
+
+% All options specified
+{myapp, "1.1.0", transient, [otherapp]}
+
+% Some options specified
+{myapp, "1.1.0"}
+{myapp, "1.1.0", transient}
+{myapp, "1.1.0", [otherapp]}
+{myapp, transient}
+{myapp, [otherapp]}
+{myapp, transient, [otherapp]}
+```
+
+In this case, `myapp` is included in the release, its app version is `"1.1.0"`, its start type is `transient`, and its list of included apps is `[otherapp]`. These options directly correspond to the [release resource file (`.rel`)](//erlang.org/doc/man/rel.html):
+
+```erlang
+ {<app_name>,       % atom()
+  <app_vsn>,        % string()
+  <start_type>,     % permanent | transient | temporary | load | none
+  <included_apps>}  % [atom()]
+```
+
+- Elements after `<app_name>` may be omitted, as long as the order is preserved.
+- If `<app_vsn>` is included (uncommon), it must match `.app.src`. Otherwise, `relx` determines it from `.app.src`.
+- The release resource file syntax, repeated below for completeness, makes it so that:
```

Comment:
I'm not quite sure whether this line is useful or is TMI to the reader. The intent is to distinguish between what `relx` does (e.g. determine `<app_vsn>`) versus what Erlang is responsible for when building the release (e.g. provide a default for `<start_type>`).